compiler/tests-spec/testData/diagnostics/notLinked/dfa/pos/30.fir.kt

```diff
@@ -11,9 +11,9 @@ fun case_1(x: Class?) {
     if (x!!.prop_8?.prop_8?.prop_8?.prop_8 != null) {
         <!DEBUG_INFO_EXPRESSION_TYPE("Class & Class?")!>x<!>
         <!DEBUG_INFO_EXPRESSION_TYPE("Class & Class?")!>x<!>.prop_8
-        <!DEBUG_INFO_EXPRESSION_TYPE("Class & Class?")!>x<!>.prop_8.<!INAPPLICABLE_CANDIDATE!>prop_8<!>
-        <!DEBUG_INFO_EXPRESSION_TYPE("Class & Class?")!>x<!>.prop_8.<!INAPPLICABLE_CANDIDATE!>prop_8<!>.<!INAPPLICABLE_CANDIDATE!>prop_8<!>
-        <!DEBUG_INFO_EXPRESSION_TYPE("Class & Class?")!>x<!>.prop_8.<!INAPPLICABLE_CANDIDATE!>prop_8<!>.<!INAPPLICABLE_CANDIDATE!>prop_8<!>.<!INAPPLICABLE_CANDIDATE!>prop_8<!>
+        <!DEBUG_INFO_EXPRESSION_TYPE("Class & Class?")!>x<!>.prop_8.<!UNSAFE_CALL!>prop_8<!>
```

Comment:
This (30) and below (31) share the same issue. Filed https://youtrack.jetbrains.com/issue/KT-44560